### PR TITLE
data: project addition by Sriparno08

### LINKED ISSUE #619: data: project addition by Sriparno Roy
### Add a new project to the list

I want to add a new project to the list and I'm in GSSOC'23.

Name: Start-Contributing
Aim: Providing Open-Source Resources

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Sriparno08**
## Related Issue

 Fixes #619 

## Description

This project aims to give beginners kickstart their open-source journey.

## Screenshots

Screenshot 2023-05-23 131045

### CODE REVIEW

File: src/DB/projects.json [hunk 1]
```diff
@@ -1096,5 +1096,23 @@
         "tech": ["Javascript", "Css", "Html"]
       }
     ]
+  },
+  {
+    "github_username": "YOUR_GITHUB_USERNAME",
```
Comment (priyankarpal) [inline src/DB/projects.json:1101]: add your github username

File: src/DB/projects.json [hunk 2]
```diff
@@ -1096,5 +1096,23 @@
...
+      "Instagram": "",
+      "YouTube": "",
+      "Twitter": "https://twitter.com/Sriparno08"
+    },
+    "Projects": [
+      {
+        "link": "https://github.com/Sriparno08/Start-Contributing",
+        "title": "Start Contributing",
+        "description": "This project aims to help beginners get started with contributing to open source by providing a substantial amount of useful resources. Everyone is welcome to contribute and be a part of this project.",
+        "tech": ""
```
Comment (priyankarpal) [inline src/DB/projects.json:1114]: add tech stacks that you have used to make this project

Comment (Sriparno08) [reply src/DB/projects.json:1114]: Hey @priyankarpal, this project is just a repository that contains open-source resources. I haven't really used anything that I can include in my tech stack. So, what should I put here?

Comment (Sriparno08) [reply src/DB/projects.json:1120]: Okay 👍

**Comment by priyankarpal** [comment]
this branch has a conflicts error. Please resolve it

**Comment by Sriparno08** [comment]
Hey, fixed the conflict. Have a look!

**Review by priyankarpal — APPROVED** [review]
Completed. Thanks for your contributions 🎉 ❤️

**Pull request merged**